salt/modules/lxc.py

```diff
@@ -727,10 +781,10 @@ def _get_base(**kwargs):
     kw_overrides = copy.deepcopy(kwargs)
 
     def select(key, default=None):
-        kw_overrides_match = kw_overrides.pop(key, None)
+        kw_overrides_match = kw_overrides.pop(key, _marker)
         profile_match = profile.pop(key, default)
```

Comment:
Another good catch. I've submitted this in https://github.com/saltstack/salt/pull/21011.

I know you don't like it, but if the bug exists in the 2015.2 branch, it needs to be fixed there so that the fix gets into 2015.2.0. I'd be more than happy to coordinate with you to get the bug fixes into the appropriate branch. The good thing is that the 2015.2 branch and develop are pretty much the same now.

I'll be around after 15:30 CET if you want to find me on Freenode.